config/server-vars.yml

```diff
@@ -235,3 +235,14 @@ COMMON_ENABLE_NEWRELIC_INFRASTRUCTURE: "%%COMMON_ENABLE_NEWRELIC_INFRASTRUCTURE%
 
 #Insights
 EDXAPP_ANALYTICS_DASHBOARD_URL: "%%EDXAPP_ANALYTICS_DASHBOARD_URL%%"
+
+# Azure keyvault configs 
+EDXAPP_GET_SECRETS_FROM_AZURE_KEYVAULT: %%EDXAPP_GET_SECRETS_FROM_AZURE_KEYVAULT%%"
+EDXAPP_AZURE_MSI_REQUEST_URI: "%%EDXAPP_AZURE_MSI_REQUEST_URI%%"
+EDXAPP_KEYVAULT_PAYLOAD: "%%EDXAPP_KEYVAULT_PAYLOAD%%"
+EDXAPP_KEYVAULT_URL: "%%EDXAPP_KEYVAULT_URL%%"
+EDXAPP_API_VERSION: "%%EDXAPP_API_VERSION%%"
```

Comment:
Suggest renaming EDXAPP_API_VERSION to something more descriptive.  Which API does this pertain to KeyVault?

Reply:
Updated.